src/components/Tag/Tag.styles.ts

```diff
@@ -64,4 +67,13 @@ export const TagStyled = styled.div<TStyleProps>`
     ${({ color, theme, tone }) =>
         theme !== 'status' && color && getColors(color, tone)}
     width: ${({ width }) => width};
+
+    ${({ width, theme }) =>
```

Comment:
if they have the same logic y not combine. It would be easier on the CSS in this case, a case so forex
`height: ${width}; width: ${width}; border-radius: 50%;`. It's less heavy on the browser this way

Reply:
good call dude, fixed